app/settings/categories/categories.html

```diff
@@ -53,9 +53,29 @@ <h2 class="listing-item-title"><a ng-href="/settings/categories/{{category.id}}"
                                 <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#trash"></use>
                             </svg>
                             <span class="hidden">Delete</span>
+                            </button>
+                    </div>
+
+                    <div id="category-{{category.id}}" class="  listing-item" ng-repeat="child in category.children | orderBy:'realname'" ng-class="{ 'warning': isToggled(category) }" ng-click="toggleCategory(category)" ng-style="{'marginLeft': '50px', 'border': 'none'}" ng-if="category.children">
```

Comment:
I don't think we should have the `orderBy` filter here. The filter above is probably an accidental copy/paste from users admin